runtime/ai/analyst_agent.go

```diff
@@ -190,7 +201,12 @@ Today's date is {{ .now.Format "Monday, January 2, 2006" }} ({{ .now.Format "200
 {{ if .explore }}
 Your goal is to analyze the contents of the dashboard "{{ .explore }}", which is powered by the metrics view "{{ .metrics_view }}".
 The user is actively viewing this dashboard, and it's what you they refer to if they use expressions like "this dashboard", "the current view", etc.
-The metrics view's definition and time range of available data has been provided in your tool calls. You should:
+The metrics view's definition and time range of available data has been provided in your tool calls.
+{{ if .time_range }}Use time range: "{{ .time_range }}"{{ end }}
+{{ if .filters }}Use filters: "{{ .filters }}"{{ end }}
```

Comment:
This is called `where` now. Also instead of "Use filters", maybe say "Use where filters:" so it's clearer that it shouldn't map them to `having`.